src/components/DevDummyLogin/index.js

```diff
@@ -14,12 +14,12 @@ import {
   View
 } from 'react-native';
 import {StackActions, useNavigation} from '@react-navigation/native';
-import {useSetRecoilState} from 'recoil';
 import {useSafeAreaInsets} from 'react-native-safe-area-context';
+import {useSetRecoilState} from 'recoil';
 
 import StorageUtils from '../../utils/storage';
 import TokenStorage from '../../utils/storage/custom/tokenStorage';
-import useProfileHook from '../../hooks/core/profile/useProfileHook';
+import useUserAuthHook from '../../hooks/core/auth/useUserAuthHook';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `useUserAuthHook` is replacing the old `useProfileHook`. Ensure that this new hook provides all the necessary functionality and state management that was previously provided by `useProfileHook`. If there are any missing functionalities, consider extending `useUserAuthHook` to include them.
```diff
- import useProfileHook from '../../hooks/core/profile/useProfileHook';
+ import useUserAuthHook from '../../hooks/core/auth/useUserAuthHook';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->